pkg/scc/systeminfo/provider.go

```diff
@@ -0,0 +1,81 @@
+package systeminfo
+
+import (
+	v3 "github.com/rancher/rancher/pkg/generated/controllers/management.cattle.io/v3"
+	"k8s.io/apimachinery/pkg/labels"
+	"net/url"
+
+	"github.com/pborman/uuid"
+	"github.com/rancher/rancher/pkg/scc/util"
+	"github.com/rancher/rancher/pkg/settings"
+	coreVersion "github.com/rancher/rancher/pkg/version"
+)
+
+const (
+	RancherProductIdentifier = "rancher"
+	RancherCPUArch           = "unknown"
+)
+
+type InfoProvider struct {
+	RancherUuid uuid.UUID
+	ClusterUuid uuid.UUID
+	nodeCache   v3.NodeCache
+}
+
+func NewInfoProvider(rancherUuid uuid.UUID, clusterUuid uuid.UUID, nodeCache v3.NodeCache) *InfoProvider {
+
+	return &InfoProvider{
+		RancherUuid: rancherUuid,
+		ClusterUuid: clusterUuid,
+		nodeCache:   nodeCache,
+	}
+}
+
+// GetVersion returns a version number for the systeminfo provider
+func GetVersion() string {
+	var version string
+	version = coreVersion.Version
+	if util.VersionIsDevBuild() {
+		// TODO: maybe SCC devs can give us a static dev version?
+		version = "2.10.3"
```

Comment:
Maybe use a version that never existed instead? 2.99.99?

Reply:
> API error: No product found on SCC for: rancher 2.99.99 unknown INTERNAL-USE-ONLY-9e4b-3166 json api/connect/v4/systems/products activate (code: 422)

Reply:
we've updated to use `other`. all good here